hexa/plugins/connector_accessmod/models.py

```diff
@@ -239,7 +231,7 @@ def owner(
         return self.project.owner
 
     def index_object(self):
```

Comment:
Maybe `index_object` could be removed altogether?

Reply:
I think it's required for the permission system :/